### ISSUE TITLE
Decide on a better way to handle uncollapsed superposed states

### ISSUE THREAD

**Issue by AgustinVallejo** [phetsims/quantum-measurement#33]
Currently, for the 'Basis state' radio buttons to toggle and untoggle freely we are using `QuantumUncollapsedCoinStateValues = [ 'up', 'down', 'superposed' ]` in addition to the normal 'up' and 'down' in `QuantumCoinStateValues`. After discussing with @jbphet we agreed that there should be a safer and more elegant way of stating that a coin is in superposition.

**Comment by jbphet** [comment]
@AgustinVallejo - I made some modifications to the general state handling in #91, and I know that you did some work on these buttons for phet-io.  Is this still an issue?  If so, let's meet and come up with a plan on how to best address it.

**Comment by AgustinVallejo** [comment]
I don't think this is an issue. There are still the three states, but we only create radio buttons for up and down, and it seems to work just fine in the sim and in phet-io, also it's readonly. I think it's safe to close
